core/cas-server-core-util/src/test/java/org/apereo/cas/util/ScriptingUtilsTests.java

```diff
@@ -22,25 +23,25 @@
 public class ScriptingUtilsTests {
 
     @Test
-    public void verifyInlineGroovyScript() {
```

Comment:
Please reset all formatting changes.

Reply:
I didn't change any formatting, I removed the unnecessary public modifiers from the test methods since CAS is now running on Junit5. I can revert them if you'd prefer to keep this PR cleaner, let me know.

Reply:
Please revert.